hoprd/hoprd/build.rs

```diff
@@ -3,6 +3,10 @@ use vergen_gix::{Emitter, GixBuilder};
 
 pub fn main() -> Result<()> {
     // Adds a short SHA hash of the commit as `VERGEN_GIT_SHA` env variable
-    let git = GixBuilder::default().sha(true).build()?;
-    Emitter::default().add_instructions(&git)?.emit()
+    if std::env::var("VERGEN_GIT_SHA").is_err() {
+        let git = GixBuilder::default().sha(true).build()?;
+        Emitter::default().add_instructions(&git)?.emit()
+    } else {
+        Ok(())
+    }
```

Comment:
What happens, if you download the source as package and there's no git?

Reply:
Unless you use nix, the build will not have any Git info and therefore the information via HTTP API will be incomplete.